code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -655,6 +655,28 @@
 	holder.remove_reagent(src.id, 0.5 * REAGENTS_METABOLISM)
 	return
 
+//Not to be confused with inaprovaline
+/datum/reagent/invaprovaline
+	name = "Invaprovaline"
+	id = "invaprovaline"
```

Comment:
You need to change this ID. Make it "inaprov_vox" or something.

This is such a minor change people are literally going to confuse it with a typo.